packages/block-library/src/table/state.js

```diff
@@ -276,16 +276,25 @@ export function deleteColumn( state, {
  * @return {Object} New table state.
  */
 export function toggleSection( state, sectionName ) {
-	// Section exists, replace it with an empty row to remove it.
+	const sectionSaved = `${ sectionName }Saved`;
+
+	// Section exists, save it and replace it with an empty row to remove it.
 	if ( ! isEmptyTableSection( state[ sectionName ] ) ) {
-		return { [ sectionName ]: [] };
+		return { [ sectionName ]: [], [ sectionSaved ]: [ ...state[ sectionName ] ] };
+	}
+
+	// Check if we already have this section
```

Comment:
Tiny thing, but part of the coding standards is to have a period at the end of comments
```suggestion
	// Check if we already have this section.
```